torch/_subclasses/fake_impls.py

```diff
@@ -362,6 +362,7 @@ def repeat_interleave_tensor(fake_mode, func, repeats, output_size=None):
     return repeats.new_empty(output_size)
 
 
+@register_op_impl(torch.ops.aten.item.default)
```

Comment:
@ezyang Is this ok? In export, we give option for user to preserve CIA ops in the inference IR. `aten.item` is one such CIA op. As a result, we need a fake rule for it while tracing.